app/models/project.rb

```diff
@@ -1,4 +1,10 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def avg_yoe_of_contestants
+    contestants.calculate(:average, :years_of_experience)
```

Comment:
you can just use the `.average` AR method vs `.calculate`. That is a cool thing that you found. 
```ruby
contestants.average(:years_of_experience).to_f
```